client/src/partials/patients/registration/registration.html

```diff
@@ -38,6 +38,7 @@
                         ng-maxlength="PatientRegCtrl.length150"
                         name="display_name"
                         ng-model="PatientRegCtrl.medical.display_name"
+                        autocomplete="off"
```

Comment:
Can you set this in `bhFormDefaults` and see if that does it?  Change 'none' to 'off'.